### ISSUE TITLE
My app suddenly couldn't use deep link shared from facebook Sharekit SDK

### ISSUE THREAD

**Issue by chunyili200556** [BranchMetrics/ios-branch-deep-linking-attribution#955]
Neither iOS and Android can use the deep link shared from Facebook share SDK , iOS will get '1003' error code.
Only facebook share SDK will get this problem , other platform worked fine.
Could you please help me to fix this problem.

**Comment by chunyili200556** [comment]
There's a interest situation.If I click the link from other apps like Line、Slack or Email , deeplink always worked fine even the link is from Safari facebook website, only Facebook app will cause this issue .And the fun part is after I click the link from facebook app ,other apps will also get the error code , unless I remove my app and reinstall.

**Comment by echo-branch** [comment]
https://blog.branch.io/how-to-deep-linking-from-facebook/

**Comment by echo-branch** [comment]
TLDR;  FB prevents deeplinks from leaving the FB app without configuring FB app links.

**Comment by echo-branch** [comment]
I think I misunderstood your issue.  

You already have app links setup and when your app opens you get an error from us?  Could you log the link that is provided to your App from FB?  FB appends extra parameters to deeplinks, it's possible we're not handling those values correctly.

**Comment by echo-branch** [comment]
Server team has informed me there is an issue with FB links currently, they are working on a fix.

**Comment by chunyili200556** [comment]
thank u so much!!!!!

**Comment by echo-branch** [comment]
Server team says they've rolled back the change that caused this issue.  Let me know if you're still encountering it.

**Comment by chunyili200556** [comment]
The problem is been solved. Thanks you for ur hard work!!!
